tests/FieldDescription/BaseFieldDescriptionTest.php

```diff
@@ -346,6 +346,20 @@ public function testGetTranslationDomain(): void
         $this->assertSame('ExtensionDomain', $description->getTranslationDomain());
     }
 
+    public function testGetTranslationDomainWithFalse(): void
+    {
+        $description = new FieldDescription('name', ['translation_domain' => false]);
+
+        $admin = $this->createMock(AdminInterface::class);
+        $description->setAdmin($admin);
+
+        $admin->expects($this->never())
+            ->method('getTranslationDomain')
+            ->willReturn('AdminDomain');
```

Comment:
Since we expect this method to never been called, not sure if `->willReturn` makes sense.